cli/Sources/TuistKit/Commands/SetupCommand.swift

```diff
@@ -0,0 +1,14 @@
+import ArgumentParser
+import Foundation
+
+struct SetupCommand: ParsableCommand {
+    static var configuration: CommandConfiguration {
```

Comment:
Didn't you mention "tuist up" as an idea? Should we move other "setup" commands like the one that we have for the registry under this one in a follow-up PR?